src/ec/packages/ec-component-search-form/search-form.story.js

```diff
@@ -0,0 +1,38 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+
+import searchFormDocs from './docs/search-form.md';
+import searchForm from './search-form.html.twig';
+
+storiesOf('Components/SearchForm', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withNotes)
+  .addDecorator(withCode)
+  .add(
+    'default',
+    () =>
+      searchForm({
+        textInput: {
+          id: 'input-search',
+          name: 'search',
+          extra_classes: 'ecl-search-form__text-input',
```

Comment:
there should be a (hidden) label for the text input. Even if it is not displayed to the users, it offers better accessibility (for screen reader for instance)

Reply:
Do you mean [placeholder](https://www.w3schools.com/tags/att_input_placeholder.asp)?

Reply:
Please check if I added it well in https://github.com/ec-europa/ecl-twig/pull/28/commits/72483ba8d0c2d29290bdd9a64af43fa2ae0d59e7

Reply:
You should add a `label` key in your `text_input` object:
```
text_input: {
...
  label: 'Search',
...
}
```